---
layout: essay
type: essay
title: Evidence of Scholarly Ability
date: 2020-12-01
labels:
  - PhD Portfolio
---

This page is to show evidence of scholarly ability, as required for the ICS [PhD portfolio](http://www.ics.hawaii.edu/academics/graduate-degree-programs/ph-d-in-ics/#phd-portfolio). This includes a literature and publications:

## Literature Review (not available yet)

[Using phylogenetic information to understand microbial contribution in microbiome studies](https://puumanamana.github.io/essays/literature-review.pdf)

## Publications

### First author
- Arisdakessian C, Poirion O, Yunits B, Zhu X, Garmire LX. DeepImpute:
  an accurate, fast, and scalable deep neural network method to impute
  single-cell RNA-seq data. Genome biology. 2019 Dec;20(1):1-4.
- Arisdakessian, C., Cleveland, S.B. and Belcaid,
  M., 2020. MetaFlow|mics: Scalable and Reproducible Nextflow
  Pipelines for the Analysis of Microbiome Marker Data. In Practice
  and Experience in Advanced Research Computing (pp. 120-124).
  
### Collaborations
- Zhu X, Wolfgruber TK, Tasato A, Arisdakessian C, Garmire DG, Garmire
  LX. Granatum: a graphical single-cell RNA-Seq analysis pipeline for
  genomics scientists. Genome medicine. 2017 Dec;9(1):1-2.
- Zhu, X., Yunits, B., Wolfgruber, T., Liu, Y., Huang, Q., Poirion,
  O., Arisdakessian, C., Zhao, T., Garmire, D. and Garmire,
  L., 2019. GranatumX: A community engaging and flexible software
  environment for single-cell analysis. bioRxiv, p.385591.
- Du, Y., Huang, Q., Arisdakessian, C. and Garmire,
  L.X., 2020. Evaluation of STAR and Kallisto on Single Cell RNA-Seq
  Data Alignment. G3: Genes, Genomes, Genetics, 10(5), pp.1775-1783.

## Under review
- Arisdakessian C, Nigro O., Steward G., Poisson G., Belcaid
  M. CoCoNet: An Efficient Deep Learning Tool for Viral Metagenome
  Binning (2020)
